numpy/_core/_add_newdocs.py

```diff
@@ -4799,7 +4799,7 @@
 
 add_newdoc('numpy._core.multiarray', 'get_handler_name',
     """
-    get_handler_name(a: ndarray) -> str,None
+    get_handler_name(a: ndarray) -> str | None
```

Comment:
sorry, I couldn't help myself

Reply:
Hehe, but if we add the `--` to make it a signature, I guess this'll break anyway :P